CHANGELOG.md

```diff
@@ -421,6 +421,10 @@ select filters)
 
 ### Added:
 
+- [Added groundwork classes for POJO `DataApiRequest` build path](https://github.com/yahoo/fili/issues/769)
+    * Generator contract, builder, and POJO object have all been added, along with some silent contract changes.
+    * Immutable implementations of `LinkedHashSet`, `LinkedHashMap`, and `ApiFilters` have been added. 
+
```

Comment:
Need some additional CHANGELOG:

I would call out in the changed section:

Deprecated Optionals on constructors and withers
Replaced OptionalInt with Optional<Integer> on some interfaces